app/components/header/header-account-custom-html.njk

```diff
@@ -0,0 +1,44 @@
+{% set html_style = "background-color: #f0f4f5;" %}
+{% set title = "Header with account (logged in, custom html)" %}
+{% from "components/header/macro.njk" import header %}
+{% extends "layout.njk" %}
+
+{% block body %}
+  <style>
+    .app-count {
+      align-self: center;
+      align-items: center;
+      background-color: #d5281b;
+      border-radius: 1em;
+      color: #fff;
+      font-size: 14px;
+      height: 1.5em;
+      line-height: 1.5em;
+      min-width: 1.5em;
+      padding: 0.25em;
+      display: inline-flex;
+      justify-content: center;
+    }
+  </style>
+
+  {{ header({
+    account: {
+      items: [
+        {
+          href: "#",
+          text: "Check your profile"
+        },
+        {
+          html: '<a class="nhsuk-header__account-link" href="#">Notifications</a> <span class="app-count">8</span>'
```

Comment:
Perhaps in this example the notification count out to be part of the link (so that it’s clickable and more accessible if the link is read out out-of-context)? Would save having to add the html for the link tag too.

```suggestion
          href: "#",
          html: 'Notifications <span class="app-count">8</span>'
```